protostar/testing/test_runner.py

```diff
@@ -193,7 +198,9 @@ async def _invoke_test_cases(
         execution_state: TestExecutionState,
     ) -> None:
         for test_case in test_suite.test_cases:
-            test_result = await self._invoke_test_case(test_case, execution_state)
+            test_result = await self._invoke_test_case(
+                test_case, execution_state
+            )
```

Comment:
```suggestion
            test_result = await self._invoke_test_case(test_case, execution_state)
```

nitpick: unnecessary change